### DISCUSSION TITLE
i have been rejected for 5times，but i check my information again and again

### DISCUSSION THREAD

**Discussion by leodamin** [community/community#54310] [GitHub Education]
### Hi everyone,

I have applied for 5 times, but all of them were rejected. I revised my homepage profile according to the guidance, changed my name to the name on my photo, and used the camera of my mobile phone to take pictures directly. However, when I submitted again, I was rejected all the time. I hope an official customer service can answer my questions and review my information
![image](https://user-images.githubusercontent.com/129604455/235814626-9ffadc32-cc4e-4328-851d-b84817064a34.jpg)

**Comment by samus-aran** [comment]
Hi @leodamin  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
